### PR TITLE
Fix nmstate crd

### PR DESCRIPTION
**Pull request by phoracek**
Fix missing pieces from nmstate component.

### CODE REVIEW

**Comment by phoracek** [comment]
@SchSeba @qinqon could you please take a look? We still miss NetworkManager on the local cluster thought.

File: data/nmstate/003-nmstated.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
```
Comment (SchSeba) [inline data/nmstate/003-nmstated.yaml]:
```
message: 'could not apply (apps/v1, Kind=DaemonSet) nmstate/nm-state-controller:
could not create (apps/v1, Kind=DaemonSet) nmstate/nm-state-controller: DaemonSet.apps
"nm-state-controller" is invalid: spec.template.metadata.labels: Invalid value:
map[string]string{"nmstate.io":"state-controller"}: `selector` does not match
template `labels`'
```

With the new version you should add the same label in the spec section

Comment (phoracek) [reply data/nmstate/003-nmstated.yaml]: Oh yeah, you are right. Thanks

**Comment by phoracek** [comment]
@qinqon we can introduce some templating, but not in this patch since it would require way more changes. Also, I'm not so sure about it since we still have to pull manifests from third-party repository.

The rename is so we create CRD first and the daemon only at the end. Not sure it matters, but in all the other components we start DS last. For the sake of consistency.

File: data/nmstate/003-nmstated.yaml (hunk 1 above)
Comment (phoracek) [reply data/nmstate/003-nmstated.yaml]: Fixed, could you please take another look?

**Comment by phoracek** [comment]
ci test please

**Comment by SchSeba** [comment]
ci test please

**Comment by phoracek** [comment]
@SchSeba could you check again?

**Pull request merged**